### PR TITLE
neutron: bump images tag

### PR DESCRIPTION
**Pull request by mnasiadka**
Includes https://github.com/stackhpc/neutron/commit/f83a97dea21c4d60cb2da16984fffcfb231fc239

### CODE REVIEW

**Review by priteau — DISMISSED** [review]

File: etc/kayobe/kolla/globals.yml [hunk 1]
```diff
@@ -9,12 +9,12 @@ enable_docker_repo: {% raw %}"{{ 'overcloud' not in group_names or ansible_facts
 
 {% if kolla_base_distro == 'centos' %}
 bifrost_tag: yoga-20230217T160618
-neutron_tag: yoga-20230309T123152
+neutron_tag: yoga-20230315T105933
 {% elif kolla_base_distro == 'rocky' %}
 bifrost_tag: yoga-20230310T194732
```
Comment (markgoddard) [inline etc/kayobe/kolla/globals.yml]: Missing rocky 9 image?

**Comment by markgoddard** [comment]
Merge conflicts

File: etc/kayobe/kolla/globals.yml (hunk 1 above)
Comment (mnasiadka) [reply etc/kayobe/kolla/globals.yml]: Oh boy, now rocky - let me build it - do we have a way of building all three at the same time with the same tag? that would probably ease some pain next time ;-)

Comment (priteau) [reply etc/kayobe/kolla/globals.yml]: The tag is generated during build so I don't think we have this capability yet 😞 At least you can easily trigger the build from the CLI with (run it three times while changing `distro`):

```
gh workflow run stackhpc-container-image-build.yml -F distro="<centos|rocky|ubuntu>" -F regexes="neutron" -r stackhpc/yoga
```

Comment (markgoddard) [reply etc/kayobe/kolla/globals.yml]: *insert response about next-gen container image build automation*

Comment (mnasiadka) [reply etc/kayobe/kolla/globals.yml]: Done

**Comment by mnasiadka** [comment]
> Merge conflicts

done

File: etc/kayobe/kolla/globals.yml [hunk 2]
```diff
@@ -9,24 +9,26 @@ enable_docker_repo: {% raw %}"{{ 'overcloud' not in group_names or ansible_facts
 
 {% if kolla_base_distro == 'centos' %}
 bifrost_tag: yoga-20230217T160618
+<<<<<<< HEAD
```
Comment (markgoddard) [inline etc/kayobe/kolla/globals.yml]: HEAD

File: etc/kayobe/kolla/globals.yml [hunk 3]
```diff
@@ -9,24 +9,26 @@ enable_docker_repo: {% raw %}"{{ 'overcloud' not in group_names or ansible_facts
...
 ironic_tag: yoga-20230316T170311
 ironic_dnsmasq_tag: yoga-20230310T170929
+neutron_tag: yoga-20230317T111905
 prometheus_node_exporter_tag: yoga-20230315T170614
 {% else %}
 bifrost_tag: yoga-20230220T184947
 blazar_tag: yoga-20230315T125441
 caso_tag: yoga-20230315T125441
-neutron_tag: yoga-20230309T123143
+neutron_tag: yoga-20230315T110042
```
Comment (markgoddard) [inline etc/kayobe/kolla/globals.yml]: nit: not alphabetical

**Review by markgoddard — COMMENTED** [review]
Add a reno?

**Review by markgoddard — DISMISSED** [review]

**Review by markgoddard — DISMISSED** [review]
Rebased on CI fixes

**Pull request closed without merging**